jobs/consul_agent_windows/monit

```diff
@@ -0,0 +1,9 @@
+{
+  "processes": [
+    {
+      "name": "consul",
+      "executable": "C:\\var\\vcap\\packages\\confab-windows\\bin\\confab.exe",
+      "args": [ "start", "", "-config-file", "/var/vcap/jobs/consul_agent_windows/confab.json", "-foreground" ]
+    }
+  ]
```

Comment:
The non-windows version runs dnsmasq which is critical to work around issues with how Consul does DNS.  How is that going to work in the windows case?